kbase-extension/static/kbase/js/util/jobLogViewer.js

```diff
@@ -882,63 +902,61 @@ define([
             fsm.newState({ mode: 'job-not-found' });
         }
 
-        var externalEventListeners = [];
```

Comment:
renamed `externalEventListeners` to `listeners`, declared the variable at the top of the file with the other vars.